src/instructlab/config/init.py

```diff
@@ -165,18 +165,19 @@ def init(
     )
 
 
-def check_if_configs_exist() -> bool:
+def check_if_configs_exist(fresh_install) -> bool:
     if exists(DEFAULTS.CONFIG_FILE):
         overwrite = click.confirm(
             f"Found {DEFAULTS.CONFIG_FILE}, do you still want to continue?"
         )
         if not overwrite:
             raise click.exceptions.Exit(0)
-    if exists(DEFAULTS.TRAIN_PROFILE_DIR):
+    if exists(DEFAULTS.TRAIN_PROFILE_DIR) and not fresh_install:
```

Comment:
do we want to add this check to the config file as well? dont know if we want to keep them consistent

Reply:
no because only the train profiles/ train profile dir is in the persistent storage creation func. The config yaml is handled only here.